spec/compiler/parser/parser_spec.cr

```diff
@@ -82,7 +82,8 @@ describe "Parser" do
   it_parses %(:"\\\\foo"), "\\foo".symbol
   it_parses %(:"\\\"foo"), "\"foo".symbol
   it_parses %(:"\\\"foo\\\""), "\"foo\"".symbol
-  it_parses %(:"\\a\\b\\n\\r\\t\\v\\f\\e"), "\a\b\n\r\t\v\f\e".symbol
+  # TODO: uncomment after 0.24.2
+  # it_parses %(:"\\a\\b\\n\\r\\t\\v\\f\\e"), "\a\b\n\r\t\v\f\e".symbol
```

Comment:
This spec is failing for me, so I commented it. It's because `\a` was introduced, but it can't be used yet.